src/google/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
Doesn't this belong in an `ndb` subdirectory (rather than top-level `src`)?  Or should it be in a separate repository altogether?

Reply:
@tseaver Check out the [history][1] for this branch. It's a "from scratch" rewrite of the [existing][2] `ndb`.

> Or should it be in a separate repository altogether?

The answer is probably, but for the next week while @theacodes is out this is the best place for me to get started.

[1]: https://github.com/GoogleCloudPlatform/google-cloud-python/commits/ndb-py3-rewrite
[2]: https://github.com/GoogleCloudPlatform/datastore-ndb-python

Reply:
Ok.  It is a little weird to have development of completely unrelated stuff going on here, but I get it.

Reply:
@tseaver I agree, though @theacodes will be back soon enough.